src/ocean/io/select/client/SelectEvent.d

```diff
@@ -87,11 +87,25 @@ public abstract class ISelectEvent : IAdvancedSelectClient
         Constructor. Creates a custom event and hooks it up to the provided
         event handler.
 
+        Supported OR-combination of `flags`, starting with Linux 2.6.27:
+          - `EFD.NONBLOCK`: Set the `O_NONBLOCK` file status flag on the new
```

Comment:
Not sure about the value of exposing `NONBLOCK` and `SEMAPHORE` here. Again, it just makes the API more complicated for no real reason (i.e. there's no demand for those features). It's a grey area, I know... there _may_ be a future use case and just exposing the standard options is consistent with the system `eventfd` function. My feeling is slightly on the side of not copying the API of `eventfd` directly, though, as our higher level wrappers (like this class) have always presented a simplified API, just containing the functionality we need.